app/src/components/submissionPage/CharactersLeftDisplay.tsx

```diff
@@ -1,6 +1,14 @@
 import MentionElements from "../../types/MentionElements";
 import GetExtraLength from "./GetExtraLength";
 import React from "react";
+import styled from "styled-components";
+
+import { HelperText } from "./SnapSubmissionStyles";
+
+const CharactersLeft = styled(HelperText)`
+    text-align: right;
+    color: #faf8f8;
```

Comment:
We should be using CSS variables for colours (to help keep them consistent and make future themeing easier).

Adam has made some changes to those that aren't currently available, so I've raised a [new card](https://trello.com/c/5ROyo1FN) for you to pick up and do when this PR is merged.